workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,260 @@
+# Multi Source Security Viewer
+
+This plugin enables you to visualize pipeline security information from multiple sources.
+
+## Setting up the plugin
+
+### For adminstrators
+
+#### Prerequisites
+
+1. Install the CI providers you want the plugin to handle the security scan outputs from.
+   The supported CI's are:
+
+- [Jenkins](../../../jenkins/plugins/jenkins-backend/README.md)
+- [Github Actions](../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#setup)
+
+#### Procedure
+
+1. Install the plugin using the following command:
+
+```
+yarn workspace app add @backstage-community/multi-source-security-viewer
+```
+
+To enable the PipelineRun list in the Security tab on the entity view page, add the following snippet in the packages/app/src/components/catalog/EntityPage.tsx.
+
+```
+/* highlight-add-next-line */
```

Comment:
Sorry, I missed this previously these tags won't work, you'll want to do a diff instead to show the changes.